Simplenote/Classes/SPSearchResultsViewController.swift

```diff
@@ -147,11 +143,11 @@ extension SPSearchResultsViewController: UITableViewDelegate {
 extension SPSearchResultsViewController: UIScrollViewDelegate {
 
     func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
-        guard let searchBar = searchController?.searchBar, searchBar.isFirstResponder else {
-            return
-        }
-
-        searchBar.resignFirstResponder()
+//        guard let searchBar = searchController?.searchBar, searchBar.isFirstResponder else {
+//            return
+//        }
+//
+//        searchBar.resignFirstResponder()
```

Comment:
I'm assuming this is commented out for a subsequent PR? 😄

Reply:
Yes sir! This whole class is scheduled out _soon_. Thank you for reviewing this PR!!